test/test_ast.py

```diff
@@ -0,0 +1,94 @@
+import ast
```

Comment:
these tests don't fit with the style of skulpt. 
Can we turn these into unit tests?